lib/active_admin/resource_controller/streaming.rb

```diff
@@ -24,14 +24,14 @@ def stream_resource(&block)
         headers["Last-Modified"] = Time.current.httpdate
 
         if ActiveAdmin.application.disable_streaming_in.include? Rails.env
-          self.response_body = block[String.new]
+          self.response_body = block[(+"")]
```

Comment:
This one I wasn't familiar with. This comes from the `Performance/UnfreezeString` cop but we don't have it enabled in our config. If you are confident with this change, then I'd be ok with it, otherwise I would suggest we disable that cop for now.

Reply:
>  If you are confident with this change, then I'd be ok with it,

Using this approach from a while, I would say that I'm quite confident of `+''` to generate unfrozen strings for the use cases I work with

Hovewer, according to https://github.com/fastruby/fast-ruby#stringdup-vs-string-code, `String.new` generates a `ASCII-8BIT` string, while `+''` generates an UTF-8 string. If this can be an issue in this context, we can disable it